Web/src/components/games/BalloonShake/GameDisplay.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { mdMin } from 'utils/media';
+import { ReactComponent as BalloonSVG } from 'svg/balloon.svg';
+
+interface IProps {
+  secondsLeft: number;
+  count: number;
+}
+
+const GameContainer = styled.div`
+  background: var(--pale-yellow);
+  position: relative;
+  overflow: hidden;
+  height: 100vh;
+  width: 100vw;
+`;
+
+const Section = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 7rem 1rem;
+  flex-direction: column;
+  align-items: center;
+
+  & > div {
+    width: ${mdMin};
+
+    @media (max-width: ${mdMin}) {
+      width: calc(100% - 0.5em);
+    }
+  }
+`;
+
+const Balloon = styled(BalloonSVG)`
+  display: flex;
+  position: absolute;
+  justify-content: center;
+  z-index: 0;
```

Comment:
maybe just set this to `z-index: -1;` so you don't need to put `z-index: 1;` everywhere!